_config.yml

```diff
@@ -34,14 +34,14 @@ navigation:
       path: index.html
     - path: about.md
     - title: Quickstart
-      url: https://github.com/xenia-project/xenia/wiki/Quickstart
+      url: https://github.com/xenia-canary/xenia-canary/wiki/Quickstart
```

Comment:
The wiki pages need to be merged into the xenia-project wiki first. 12th generation Core is an excessively high requirement though, I think.

Reply:
That would require this branch being moved to master: https://github.com/xenia-canary/xenia-canary/tree/wiki